x-pack/plugins/uptime/public/lib/alert_types/monitor_status.tsx

```diff
@@ -31,6 +31,9 @@ export const initMonitorStatusAlertType: AlertTypeInitializer = ({
   ),
   description,
   iconClass: 'uptimeApp',
+  documentationUrl(docLinks) {
+    return `${docLinks.ELASTIC_WEBSITE_URL}guide/en/observability/${docLinks.DOC_LINK_VERSION}/monitor-status-alert.html`;
```

Comment:
https://www.elastic.co/guide/en/uptime/current/uptime-alerting.html

Reply:
Fixed in https://github.com/elastic/kibana/pull/81526/commits/3e62a8c311f500a05a4e9e1f656f207345ab3480.

Did you want me to fix the doc links for TLS as well or is the current link ok? (https://www.elastic.co/guide/en/observability/master/tls-certificate-alert.html)

Reply:
@shahzad31 ^^